lib/route/origin_security.go

```diff
@@ -34,11 +31,6 @@ func OriginSecurity(domains []Domain) SecurityHandler {
 				return
 			}
 
-			if len(origin) == 0 {
-				logger.Debug("Could not infer request origin since Origin and Referer headers are both missing")
```

Comment:
We could pass a logger through here, but on reflection these seem like debug printfs and it would probably be better to test these conditions rather than log them.